build/media_source/system/js/fields/joomla-image-select.w-c.es6.js

```diff
@@ -162,8 +166,16 @@
 
           Joomla.editors.instances[editor].replaceSelection(imageElement);
         } else {
-          const val = appendParam(Joomla.selectedMediaFile.url, 'joomla_image_width', Joomla.selectedMediaFile.width);
-          editor.value = appendParam(val, 'joomla_image_height', Joomla.selectedMediaFile.height);
+          let val = '';
+
+          if (isLocalAdapter) {
+            val = appendParam(Joomla.selectedMediaFile.url, 'joomla_image_width', Joomla.selectedMediaFile.width)
```

Comment:
```suggestion
            val = appendParam(Joomla.selectedMediaFile.url, 'joomla_image_width', Joomla.selectedMediaFile.width);
```
Eslint complains about missing semicolon, and I think he (or she?) is right ;-)